cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -62,4 +69,3 @@ spec:
           medium: {{ .Values.packageCache.medium }}
           sizeLimit: {{ .Values.packageCache.sizeLimit }}
         {{- end }}
-        
```

Comment:
nitpick: could you re-establish this newline?

Reply:
This was to remove about 10 spaces rather than removing any line or line break. Let me know if you still want me to revert it back.

Reply:
Also for that matter there are some inconsistencies in line-ending of `template/*.yaml` files. For instance `rbac-manager-clusterrole.yaml` has one trailing empty line, but for example neither `rbac-manager-clusterrolebinding.yaml` nor `rbac-manager-serviceaccount.yaml` haven't.

Reply:
Ah I see.. I kind of like the single trailing newline as a standard, what do you think?

Reply:
I'm hugely in favor of this. Actually I always add `.editorconfig` file (based on https://editorconfig.org/) to my projects to unify and semi-enforce this. Something like:

```toml
# http://editorconfig.org

# this file is the top-most editorconfig file
root = true

# all files
[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 4
trim_trailing_whitespace = true
insert_final_newline = true

[*.go]
indent_style = tab
indent_size = 4

[*.md]
trim_trailing_whitespace = false

[*.{yaml,yml}]
indent_size = 2

[Makefile]
indent_style = tab
```

If you agree with this I can open PRs in core repos.

Reply:
Nice! I would be a fan, @negz do you have any thoughts on this?